src/models.py

```diff
@@ -119,3 +104,52 @@ def callbacks_unet(callbacks_config):
     return CallbackList(
         callbacks=[experiment_timing, training_monitor, validation_monitor,
                    model_checkpoints, lr_scheduler, neptune_monitor, early_stopping])
+
+
+def mixed_dice_cross_entropy_loss(output, target, dice_weight=0.5, dice_loss=None,
```

Comment:
@taraspiotr I think instead of creating this function it is beter to pass 2 functions to the self.loss_function list of tuples

Reply:
@jakubczakon As far as I remember, `self.loss` function expects as many elements in a list, as number of output legs from UNet. Here we have one leg, so one loss function is needed, right?

Reply:
@apyskir I was thinking the same thing

Reply:
@taraspiotr @apyskir correct. nvmd